view/src/pages/home.js

```diff
@@ -189,7 +371,30 @@ class Home extends Component {
             </List>
           </Drawer>
 
-          <div>{this.state.render ? <Account /> : <Todo />}</div>
+          <div className={classes.contentContainer}>
+            {/* The alert component that is activated by any child of the home page */}
+            <Alert
+              open={this.state.showAlert}
+              handleOpen={this.openAlert}
+              handleClose={this.closeAlert}
+              severity={this.state.severity}
+              message={this.state.message}
+            />
+            {/* Load in the content of the selected component page */}
+            {this.state.toRender === "deals" ? (
+              <Deal alert={this.alert} />
+            ) : this.state.toRender === "account" ? (
+              <Account alert={this.alert} />
+            ) : this.state.toRender === "produce" ? (
+              <Produce alert={this.alert} />
+            ) : this.state.toRender === "surplus" ? (
+              <Surplus alert={this.alert} main={true} />
+            ) : this.state.toRender === "farms" ? (
+              <Farms alert={this.alert} />
+            ) : (
+              <FoodBanks alert={this.alert} main={true} />
+            )}
```

Comment:
I wonder if a switch statement would be more readable? I'm not used to seeing the ternary operator chained like this. 

I'll admit I'm biased against ternary, even when it's the better choice. So I could certainly be wrong here.